test/e2e/resize_nodes.go

```diff
@@ -44,6 +44,10 @@ const (
 )
 
 func resizeGroup(size int) error {
+	if testContext.ReportDir != "" {
+		CoreDump(testContext.ReportDir)
+		defer CoreDump(testContext.ReportDir)
```

Comment:
does this actually work? or does it immediately call CoreDump when the `if` block ends?